README.adoc

```diff
@@ -278,14 +256,16 @@ Results :
 Tests run: 3, Failures: 0, Errors: 0, Skipped: 0
 ----
 
-To see whether the tests detect a failure, go to the `MetricsTest.java` file and change the assertion values in the `validateMetric()` method. Rerun the Maven build. A test failure occurs.
+To see whether the tests detect a failure, go to the `MetricsTest.java` file and change an assertion
+in the `validateMetric()` method. Rerun the Maven build. A test failure occurs.
```

Comment:
Should we specify which assertion the user needs to change? We just say "an assertion" right now. What if we said, "the <specific assertion name here> assertion"?